src/algebra/polynomial/big_operators.lean

```diff
@@ -155,4 +155,21 @@ lemma leading_coeff_prod :
 by { rw ← leading_coeff_hom_apply, apply monoid_hom.map_prod }
 
 end no_zero_divisors
+
+section nontrivial_no_zero_divisors
```

Comment:
I think you should just put this in the previous section - there are already other lemmas in that section using non_trivial

Reply:
Yes, you're right, I'll do this.

Reply:
Ok done